damus/Views/Relays/RelayConfigView.swift

```diff
@@ -9,24 +9,55 @@ import SwiftUI
 
 struct RelayConfigView: View {
     let state: DamusState
+    
     @State var new_relay: String = ""
     @State var relays: [RelayDescriptor]
     @State private var showActionButtons = false
     
+    @StateObject var profile: ProfileModel
+    
     @Environment(\.dismiss) var dismiss
     
-    init(state: DamusState) {
+    init(state: DamusState, profile: ProfileModel) {
         self.state = state
         _relays = State(initialValue: state.pool.our_descriptors)
+        self._profile = StateObject(wrappedValue: profile)
     }
     
-    var recommended: [RelayDescriptor] {
-        let rs: [RelayDescriptor] = []
-        return BOOTSTRAP_RELAYS.reduce(into: rs) { xs, x in
-            if state.pool.get_relay(x) == nil, let url = RelayURL(x) {
-                xs.append(RelayDescriptor(url: url, info: .rw))
+    var recommended: [(RelayDescriptor, Int)] {
+        //var suggested_relays: [RelayDescriptor] = []
+        var suggested_relays: [(RelayDescriptor, Int)] = [] // The Int is to keep count of how many people we follow us this relay.
+        
+        if let profile_contacts = profile.contacts {
+            let contacts_pubkeys = profile_contacts.referenced_pubkeys.map { $0.ref_id }
+            let following = FollowingModel(damus_state: state, contacts: contacts_pubkeys)
```

Comment:
Why do you need the `FollowingModel`? It looks like you're just passing in pubkeys and then reading the same ones back out?

So maybe just:
```swift
for pubkey in contacts_pubkeys { ... }
```

Reply:
Removed.